client/src/components/Input.js

```diff
@@ -31,6 +32,7 @@ const ButtonIcon = styled.button`
   height: 20px;
   width: 20px;
   border: none;
+  padding: 0px;
```

Comment:
You could add `padding: 0;` to your GlobalStyles as well and just add padding where you need it.

Reply:
Good idea. Its much easier now to style the components 😉